lib/secrets.rb

```diff
@@ -158,4 +161,8 @@ def read_vault_token(authfile_path)
   rescue OpenSSL::X509::CertificateError
     File.read(authfile_path)
   end
+
+  def log(msg)
+    puts "#{Time.now.to_i}: #{msg}" unless ENV["testing"]
```

Comment:
to_i seems weird ... can we have a real date format ?
aren't our logs supposed to have dates automatically ... maybe we should add that instead ?